src/AElf.OS/OSAElfModule.cs

```diff
@@ -25,7 +26,20 @@ public override void ConfigureServices(ServiceConfigurationContext context)
     {
         var configuration = context.Services.GetConfiguration();
 
-        Configure<AccountOptions>(configuration.GetSection("Account"));
+        var address = Environment.GetEnvironmentVariable("AELF_ADDRESS");
+        var password = Environment.GetEnvironmentVariable("AELF_PASSWORD"); 
+        if (string.IsNullOrEmpty(address)) 
+        { 
+            Configure<AccountOptions>(configuration.GetSection("Account"));
+        }
+        else 
+        { 
+            Configure<AccountOptions>(option => 
+            { 
+                option.NodeAccount = address; 
+                option.NodeAccountPassword = password;
+            });
+        }
```

Comment:
Why do we need this customization?

Reply:
For the same reason above. Now we install aelf-command and create account in the docker container, so every time we get a new private key, we need to set it as an environment variable so that load it when the chain starts.